benchmarks/000.microbenchmarks/020.network-benchmark/python/function.py

```diff
@@ -42,14 +58,12 @@ def handler(event):
         server_socket.settimeout(2)
     server_socket.close()
    
-    if consecutive_failures != 5:
-        with open('/tmp/data.csv', 'w', newline='') as csvfile:
-            writer = csv.writer(csvfile, delimiter=',')
-            writer.writerow(["id", "client_send", "client_rcv"]) 
-            for row in times:
-                writer.writerow(row)
-      
-        client = storage.storage.get_instance()
-        key = client.upload(output_bucket, 'results-{}.csv'.format(request_id), '/tmp/data.csv')
-
-    return { 'result': key }
+    with open('/tmp/data.csv', 'w', newline='') as csvfile:
+        writer = csv.writer(csvfile, delimiter=',')
+        writer.writerow(["id", "client_send", "client_rcv"]) 
+        for row in times:
+            writer.writerow(row)
+    
+    client = storage.storage.get_instance()
+    key = client.upload(output_bucket, f'results-{request_id}.csv', '/tmp/data.csv') 
+    return { 'status': 'success', 'result': 'Returned with no error', 'measurement': key }
```

Comment:
This is correct, but we also need to update the experiment in `sebs/experiments/...` to make sure it now checks for `measurement` key to download results.